spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 describe SignUp::CompletionsController do
   describe '#show' do
+    render_views
```

Comment:
`render_views` slows down the tests, can we enable this for just the few tests we need it in?

Reply:
yep, done in cd6ffd3. rspec makes it difficult since you can't specify `render_views` on a specific test, only on a group.